content/docs/addons-test-utils.md

```diff
@@ -298,20 +298,19 @@ Same as [`scryRenderedComponentsWithType()`](#scryrenderedcomponentswithtype) bu
 renderIntoDocument(element)
 ```
 
-Render a React element into a detached DOM node in the document. **This function requires a DOM.** It is effectively equivalent to:
+दस्तावेज़ में एक अलग DOM नोड में एक React एलिमेंट रेंडर करें। **इस फ़ंक्शन के लिए DOM की आवश्यकता है।** यह प्रभावी रूप से इसके बराबर है:
```

Comment:
revisit grammar of first sentence

Reply:
fixed in c7978c039248e471aa5a0981d394758e8879d432